guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -33,64 +31,30 @@ For more information, see {InstallingSmartProxyDocURL}installing-capsule-server-
 If you set a wildcard value `*` for the certificate's Common Name `CN =` in the `/root/{context}_cert/openssl.cnf` configuration file, you must add the `-t {certs-proxy-context}` option to the `katello-certs-check` command.
 +
 If the command is successful, it returns two `{certs-generate}` commands, one of which you must use to generate the certificate archive file for your {SmartProxyServer}.
-ifdef::satellite[]
-+
-.Example output of `katello-certs-check`
-[options="nowrap", subs="+quotes,attributes"]
-----
-Validation succeeded.
-
-To use them inside a NEW $CAPSULE, run this command:
-
-{certs-generate} --foreman-proxy-fqdn "$CAPSULE" \
-    --certs-tar  "~/$CAPSULE-certs.tar" \
-    --server-cert "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem_" \
-    --server-key "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_" \
-    --server-ca-cert "_/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_" \
-
-To use them inside an EXISTING $CAPSULE, run this command INSTEAD:
-
-  {certs-generate} --foreman-proxy-fqdn "$CAPSULE" \
-    --certs-tar "~/$CAPSULE-certs.tar" \
-    --server-cert "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem_" \
-    --server-key "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_" \
-    --server-ca-cert "_/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_" \
-    --certs-update-server
-----
-endif::[]
-
-ifndef::satellite[]
 +
 .Example output of `katello-certs-check`
 [options="nowrap", subs="+quotes,attributes"]
 ----
 Validation succeeded.
 
-To use them inside a NEW $FOREMAN_PROXY, run this command:
```

Comment:
Late to the party, but this heading says `Example output` and the example output is `a NEW $FOREMAN_PROXY` and for Satellite it's branded as `a NEW $CAPSULE`. I think this deserved its own attribute.